Source/WebKit/WPEPlatform/wpe/wayland/WPEWaylandCursor.cpp

```diff
@@ -31,10 +31,19 @@
 
 namespace WPE {
 
-WaylandCursor::WaylandCursor(WPEDisplayWayland* display)
+std::unique_ptr<WaylandCursor> WaylandCursor::create(WPEDisplayWayland* display)
+{
+    auto theme = WaylandCursorTheme::create(wpe_display_wayland_get_wl_shm(display));
+    if (theme)
+        return makeUnique<WaylandCursor>(display, WTFMove(theme));
```

Comment:
Typically we write this kind of thing with the assignment inside the `if`:

```suggestion
    if (auto theme = WaylandCursorTheme::create(wpe_display_wayland_get_wl_shm(display)))
        return makeUnique<WaylandCursor>(display, WTFMove(theme));
```